Source/Core/PlaneGeometry.js

```diff
@@ -0,0 +1,384 @@
+define([
+    './BoundingSphere',
+    './Cartesian3',
+    './Check',
+    './ComponentDatatype',
+    './defaultValue',
+    './defined',
+    './Geometry',
+    './GeometryAttribute',
+    './GeometryAttributes',
+    './PrimitiveType',
+    './VertexFormat'
+], function(
+    BoundingSphere,
+    Cartesian3,
+    Check,
+    ComponentDatatype,
+    defaultValue,
+    defined,
+    Geometry,
+    GeometryAttribute,
+    GeometryAttributes,
+    PrimitiveType,
+    VertexFormat) {
+    'use strict';
+
+    /**
+     * Describes a plane by a normal and distance from the origin in world coordinates.
```

Comment:
This doesn't describe a plane; it describes the geometry for a plane.

A plane is infinite - this geometry is not - this reference doc should describe the size of the plane and - no pun intended - what plane it lies in and what direction its normal points.